src/lib/sql/sql_pipeline.cpp

```diff
@@ -2,22 +2,25 @@
 
 #include "SQLParser.h"
 #include "sql_pipeline.hpp"
+#include "sql_pipeline_control_block.hpp"
 
 namespace opossum {
 
-SQLPipeline::SQLPipeline(const std::string& sql, bool use_mvcc) : SQLPipeline(sql, nullptr, use_mvcc) {}
+SQLPipeline::SQLPipeline(const std::string& sql, ChunkUseMvcc use_mvcc) : SQLPipeline(sql, nullptr, use_mvcc) {}
```

Comment:
What does the SQL pipeline have to with chunks? If we want to use it here, it needs a different name.

Reply:
Yep, let's change that name. `enum class UseMvcc {}`?

Reply:
works for me

Reply:
done